examples/rmg/superminimal/input.py

```diff
@@ -0,0 +1,55 @@
+# Data sources
+database(
+    thermoLibraries = ['primaryThermoLibrary'],
+    reactionLibraries = [],
+    seedMechanisms = [],
+    kineticsDepositories = ['training'],
+    kineticsFamilies = 'default',
```

Comment:
I'm sure you could make it even faster by not loading all the reaction families (isn't "averaging up" a slow point?)....   🏃‍♀️💨  🥇